docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -73,39 +74,39 @@ import WarpStreamSVG from "../../images/logos/warpstream.svg";
-|Confluent Cloud|<ConfluentSVG style={{width: '3rem'}} />|Streaming|Unlock the combined power of Confluent and ClickHouse Cloud through our direct integration.|
 |Apache Kafka|<KafkaSVG style={{width: '3rem', 'height': '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from Apache Kafka into ClickHouse Cloud.|
+|Confluent Cloud|<ConfluentSVG style={{width: '3rem'}} />|Streaming|Unlock the combined power of Confluent and ClickHouse Cloud through our direct integration.|
 |AWS MSK|<MskSVG style={{width: '3rem', 'height': '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from AWS MSK into ClickHouse Cloud.|
-|AzureEventHubs|<AzureEventHubsSVG style={{width: '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from Azure Event Hubs into ClickHouse Cloud.|
+|Azure Event Hubs|<AzureEventHubsSVG style={{width: '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from Azure Event Hubs into ClickHouse Cloud.|
 |Upstash|<UpstashSVG style={{width: '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from Upstash into ClickHouse Cloud.|
 |WarpStream|<WarpStreamSVG style={{width: '3rem'}} />|Streaming|Configure ClickPipes and start ingesting streaming data from WarpStream into ClickHouse Cloud.|
+|Amazon S3|<S3SVG style={{width: '3rem', height: 'auto'}} />|Object Storage|Configure ClickPipes to ingest large volumes of data from object storage.|
 
 More connectors are will get added to ClickPipes, you can find out more by [contacting us](https://clickhouse.com/company/contact?loc=clickpipes).
 
 ## Supported data formats
 
 The supported formats are:
 
-| Format                                                                                    | Support     |
-|-------------------------------------------------------------------------------------------|-------------|
-| [JSON](../../../interfaces/formats.md/#json)                                               | ✔           |
-| [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent)                |*Coming Soon*|
-| [TabSeparated](../../../interfaces/formats.md/#tabseparated)                               |*Coming Soon*|
-| [CSV](../../../interfaces/formats.md/#csv)                                                 |*Coming Soon*|
+| Format| Kafka Streaming | Object Storage |
+|-------|-------|----------------|
+| [JSON](../../../interfaces/formats.md/#json)                               | ✔           | ✔           |
+| [CSV](../../../interfaces/formats.md/#csv)                                 |*Coming Soon*| ✔           |
+| [TabSeparated](../../../interfaces/formats.md/#tabseparated)               |*Coming Soon*| ✔           |
+| [Parquet](../../../interfaces/formats.md/#parquet)                         |❌            | ✔           |
+| [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent)|*Coming Soon*| ❌           |
```

Comment:
S3 connector can support other data types. Should we add a CTA to contact us if a necessary format is missing?

Reply:
I think so because it hasn't been tested and the way it is programmed we may struggle with schema inference converting from one type to JSON.

Reply:
The only reason we should not mention other formats is we haven't covered them with tests. ClickPipes for S3 just relies on CH formats supported.

> and the way it is programmed we may struggle with schema inference converting from one type to JSON.

This is not true.